Scarb.toml

```diff
@@ -4,14 +4,15 @@ version = "0.1.0"
 edition = "2023_10"
 
 [dependencies]
-starknet = "2.7.0"
+starknet = "2.8.4   "
 stark_vrf = { git = "https://github.com/dojoengine/stark-vrf.git" }
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "a13bae3" }
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.26.0" }
+openzeppelin_access = { git = "https://github.com/openzeppelin/cairo-contracts", tag = "v0.18.0"}
+openzeppelin_upgrades = { git = "https://github.com/openzeppelin/cairo-contracts", tag = "v0.18.0"}
+snforge_std = "0.31.0"
 
 [dev-dependencies]
-openzeppelin_testing = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "a13bae3" }
-openzeppelin_utils = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "a13bae3" }
+openzeppelin_testing = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.18.0"}
+openzeppelin_utils = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.18.0"}
```

Comment:
We shouldn't use tag, could we switch to `scarbs.xyz`? This eases how the versions are managed for packages using different version of it.

```toml
openzeppelin_access = "v0.18.0"
```